packages/react-button/src/components/Button/Button.types.tsx

```diff
@@ -82,3 +93,68 @@ export type ButtonSlotProps = {
 
 export interface ButtonOptions
   extends ComposeOptions<ButtonProps, ButtonSlots, ButtonSlotProps, ComposeStandardStatics> {}
+
+export interface ButtonTokens {
+  /* sizing */
+  padding: string;
+  margin: string;
+  height: string;
+  minWidth: string;
+  maxWidth: string;
+
+  contentGap: string;
+  iconSize: string;
+
+  borderRadius: string;
+  borderWidth: string;
+
+  size: {
+    smallest: string;
+    smaller: string;
+    small: string;
+    regular: string;
+    large: string;
+    larger: string;
+    largest: string;
+  };
+
+  transform: {
+    default: string;
+    hovered: string;
+    pressed: string;
+  };
+
+  transition: string;
+
+  background: TokenColorSet;
+  contentColor: TokenColorSet;
+  iconColor: TokenColorSet;
+  borderColor: TokenColorSet;
+  focusColor: TokenColorSet;
+  focusInnerColor: TokenColorSet;
+
+  accent: {
+    background: TokenColorSet;
+    contentColor: TokenColorSet;
+    iconColor: TokenColorSet;
+    borderColor: TokenColorSet;
+    focusColor: TokenColorSet;
+    focusInnerColor: TokenColorSet;
+  };
+
+  fontFamily: string;
+  fontSize: string;
+  fontWeight: string;
+}
+
+export type TokenColorSet = {
+  default: string;
+  hovered: string;
+  focused: string;
+  pressed: string;
+  checked: string;
+  checkedHovered: string;
+  checkedFocused: string;
+  checkedPressed: string;
+  disabled: string;
+};
```

Comment:
Same with this, it should probably go in the `compose` layer.

Reply:
probably

Reply:
not `compose` but probably base theme

Reply:
moved to compose